blamo/lib/Export/index.dart

```diff
@@ -43,21 +55,110 @@ class _ExportPageState extends State<ExportPage> {
           backgroundColor: Colors.deepOrangeAccent
       ),
       body: Center(
-        child: Center(child:_exportList(currentState)),
+        child: Center(child:_exportList(currentState,pickedDoc)),
       ),
     );
   }
 
-
-  Widget _exportList(StateData currentState) => ListView(
+  Widget _exportList(StateData currentState,String pickedPath) => ListView(
     children: [
-      Center(child:_tile('Email',Icons.email)),
-      Divider(),
       Center(child:_tile('Box',Icons.cloud_upload)),
       Divider(),
-      Center(child:_pdfTile('PDF',Icons.picture_as_pdf)),
+      Center(child:_pdfTile('Save ' + currentState.currentDocument + ' as PDF', Icons.picture_as_pdf)),
+      Divider(),
+      Center(child:_csvTile('Save ' + currentState.currentDocument + ' as CSV',Icons.pie_chart)),
       Divider(),
-      Center(child:_csvTile('CSV',Icons.pie_chart))
+      Center(child: _emailTile('Send Email', Icons.email, context, pickedPath)),
```

Comment:
Will this tile do something that the send email button below it doesn't?

Reply:
The tile is just a temp header right now it illustrate you sending an email with options below. As stated in the UI needs to be reworked regardless with the project wrapper and setup entirely.